src/styles/linkWithArrow.module.scss

```diff
@@ -0,0 +1,20 @@
+.link-with-arrow {
+  color: var.$color-primary--dark;
+  font-size: var.$fs-base;
+  display: flex;
+  flex-direction: row;
```

Comment:
nitpicking but I like ordering the items alphabetically 😆  makes it easier to find specific properties when there are a ton